src/components/Welcome.tsx

```diff
@@ -1,16 +1,23 @@
+import Erc20BalanceChecker from "./Erc20BalanceChecker";
+
 const Welcome = () => {
   return (
     <div
       id="Welcome"
       className="gap-3 grid grid-cols-1 lg:grid-cols-4 pt-60 p-5 pb-20"
     >
-      <div></div>
       <div className="col-span-2">
         <div className="md:text-6xl text-4xl text-black text-center">
           Welcome!
         </div>
+        WETH mainnet balance:
+        <Erc20BalanceChecker
+          erc20Address={"0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2"}
```

Comment:
May aswell use DLTx address :)

```suggestion
          erc20Address={"0xfa0B65413E3E81FAF7321f85b2AfdD0EffF13Ef5"}
```

Reply:
That address is the WETH address, not the input of balanceOf(). So I think the dltx address wouldn't work as it is not an ERC20? It checks the balance of the connected user. I added this component just to showcase typechain.

Reply:
Ok, ill let you make the call and resolve this.  Was just an idea :)